src/figuresClasses.ts

```diff
@@ -1,11 +1,76 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
```

Comment:
you can create the custom type for shape and color
```suggestion
  shape: Shape;
```